src/coreclr/src/jit/importer.cpp

```diff
@@ -3520,6 +3520,18 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
             {
                 GenTree* hwintrinsic = impHWIntrinsic(ni, clsHnd, method, sig, mustExpand);
 
+#ifdef TARGET_XARCH
```

Comment:
The same should be valid for ARM64, right?

The actual hardware intrinsic is a bit different, but that shouldn't stop constant folder for `BitOperations`